kmip/services/server/engine.py

```diff
@@ -1299,6 +1324,27 @@ def _process_register(self, payload):
 
         return response_payload
 
+    @_kmip_version_supported('1.2')
```

Comment:
Th reason why I did this is that I found in 1.0, there is no "object group member" in the request. So do we want to change it to be 1.1 instead of 1.0?